### ISSUE TITLE
[Bug]: Some icons do not appear on non-GNOME (Wayfire)

### ISSUE THREAD

**Issue by theamazing0** [GeopJr/Tuba#972]
### Describe the bug

I'm using the Flatpak version of Tuba on the Fedora Sway spin, but my window manager is set to Wayfire. Some of the icons do not appear, and instead some error icon is appearing. I'm not using any theming methods.

### Steps To Reproduce

1. Install the Tuba Flatpak
2. View the various screens to see missing icons

### Logs and/or Screenshots

![Untitled-2024-06-05-1822](https://github.com/GeopJr/Tuba/assets/66532288/c879ffc5-ac2d-49ec-ba4c-3548ab60949a)
![Untitled-2024-06-05-1822(2)](https://github.com/GeopJr/Tuba/assets/66532288/b969c70b-8fe4-4bad-8d0f-55927b285916)

### Instance Backend

Mastodon

### Operating System

Fedora Sway, using Wayfire

### Package

Flatpak

### Troubleshooting information

os: GNOME 46 (Flatpak runtime)
prefix: /app
flatpak: true
version: 0.7.2 (production)
gtk: 4.14.4 (4.14.1)
libadwaita: 1.5.1 (1.5.0)
libsoup: 3.4.4 (3.4.4)
libgtksourceview: 5.12.0 (5.12.0)

**Comment by GeopJr** [comment]
It looks like you are either missing an icon theme or have unset it. The missing icons follow the xdg spec.

My guess is that Fedora includes adwaita icon theme, so in case it has been unset, you can set it back to adwaita using `gsettings set org.gnome.desktop.interface icon-theme 'Adwaita'` (you can replace 'Adwaita' with any other icon theme).

Is Tuba the only app affected? That's something that should affect *all* your apps

**Comment by theamazing0** [comment]
Hmm actually I think you are right, sorry. Similar issues seem to be happening in other libadwaita flatpak apps. I have ran the command, and I don't have any other icon themes installed. But the icons still aren't showing.

**Comment by GeopJr** [comment]
Did you fix it? Would you mind commenting what was wrong, in case someone else encounters it in the future?

**Comment by theamazing0** [comment]
Well... So I ran your command, I checked that the icons were installed, I checked that xdg-desktop-portal-gtk was installed, I logged out and logged back in. None of that did anything. Then I ran `dbus-update-activation-environment DISPLAY` because something else wasn't working... and it fixed it? And you were correct, it wasn't just Tuba, other libadwaita apps seemed to have this issue too with certain icons.
